doc/content/en/docs/++version++/Specification/_index.md

```diff
@@ -180,9 +180,9 @@ For example, 16-byte quantity may be declared with:
 ```
 
 ### Names {#names}
-Record, enums and fixed are named types. Each has a fullname that is composed of two parts; a name and a namespace, separated by a dot. Equality of names is defined on the fullname.
+Record, enums and fixed are named types. Each has a fullname that is composed of two parts; a name and a namespace, separated by a dot. Equality of names is defined on the fullname: it is an error to specify two different types with the same name.
```

Comment:
```suggestion
Record, enums and fixed are named types. Each has a fullname that is composed of two parts: a name and a namespace, separated by a dot. Equality of names is defined on the fullname–it is an error to specify two different types with the same name.
```

Reply:
I'd add spaces before and after the dash in @kojiromike's suggestion: `... fullname - it ...`